client/app/queue/SelectConferenceTypeRadioField.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import { css } from 'glamor';
+
+import RadioField from '../components/RadioField';
+import COPY from '../../COPY';
+
+const radioOptions = [
+  { displayText: 'Pexip',
+    value: '1'},
+  { displayText: 'Webex',
+    value: '2'}
+];
+
+const SelectConferenceTypeRadioField = ({name}) => {
+  const [value, setValue] = useState("1")
+
+  return (
+    <div>
```

Comment:
```suggestion
    <>
```

You could use a React.Fragment here if you'd like.